src/main/java/com/fasterxml/jackson/databind/DefaultCacheProvider.java

```diff
@@ -0,0 +1,63 @@
+package com.fasterxml.jackson.databind;
+
+import com.fasterxml.jackson.databind.util.LookupCache;
+
+/**
+ * Container for {@link LookupCache} instances to use to override default cache implementations used.
```

Comment:
Also mention this is just the default implementation (alternates can be created).